src/core/Slider/Slider.test.tsx

```diff
@@ -0,0 +1,817 @@
+  expect(thumb.getAttribute('aria-valuenow')).toEqual('35');
+
+  act(() => {
+    fireEvent.keyDown(thumb, { key: 'ArrowRight' });
+  });
+  thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb.getAttribute('aria-valuenow')).toEqual('40');
+  act(() => {
+    fireEvent.keyDown(thumb, { key: 'Home' });
+  });
+  thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb.getAttribute('aria-valuenow')).toEqual('0');
+  act(() => {
+    fireEvent.keyDown(thumb, { key: 'End' });
+  });
+  thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb.getAttribute('aria-valuenow')).toEqual('100');
+  expect(handleOnChange).toHaveBeenCalledTimes(4);
+
+  // triggering an update with same value should not trigger callback
+  act(() => {
+    fireEvent.keyDown(thumb, { key: 'End' });
+  });
+  expect(handleOnChange).toHaveBeenCalledTimes(4);
+});
+
+it('should NOT process keystrokes when slider is disabled', () => {
+  const wrapper = render(<Slider values={[50]} step={5} disabled />);
+  const { container } = wrapper;
+  assertBaseElement(container);
+  let thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb).toBeTruthy();
+  expect(thumb.classList.contains('iui-active')).toBeFalsy();
+
+  // force thumb to have focus
+  thumb.focus();
+  expect(thumb.getAttribute('aria-valuenow')).toEqual('50');
+
+  act(() => {
+    fireEvent.keyDown(thumb, { key: 'ArrowLeft' });
+  });
+  thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb.getAttribute('aria-valuenow')).toEqual('50');
+});
+
+it('should show tooltip on thumb hover', () => {
+  const wrapper = render(<Slider values={defaultSingleValue} />);
+  const { container } = wrapper;
+  assertBaseElement(container);
+  let thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb).toBeTruthy();
+  expect(thumb.classList.contains('iui-active')).toBeFalsy();
+  expect(container.querySelector('.iui-tooltip')).toBeFalsy();
+  act(() => {
+    fireEvent.mouseEnter(thumb);
+  });
+  expect(
+    (container.querySelector('.iui-tooltip') as HTMLDivElement).textContent,
+  ).toBe('50');
+  act(() => {
+    thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+    fireEvent.mouseLeave(thumb);
+  });
+});
+
+it('should show tooltip on thumb focus', async () => {
+  const wrapper = render(<Slider values={defaultSingleValue} />);
+  const { container } = wrapper;
+  assertBaseElement(container);
+  let thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb).toBeTruthy();
+  expect(thumb.classList.contains('iui-active')).toBeFalsy();
+  expect(container.querySelector('.iui-tooltip')).toBeFalsy();
+  act(() => {
+    thumb.focus();
+  });
+  expect(
+    (container.querySelector('.iui-tooltip') as HTMLDivElement).textContent,
+  ).toBe('50');
+
+  await act(async () => {
```

Comment:
await needed here?

I'm guessing you added it in case there was tooltip animation